device-data/types/bolus/dual-square.md

```diff
@@ -60,7 +60,7 @@ We plan to migrate all Tidepool data to use `combo` as the value of this sub-typ
 		platform: yes
 	Numerical type: Floating point value rounded to the appropriate significant figures for the device's precision.
 	Range:
-		min: > 0.0
+		min: >= 0.0
```

Comment:
Did you look at the docs about which parts are editable? I can certainly iterate on how we call out the editable portions, but basically the assumption should be that everything but paragraphs of commentary is *not* editable.

The whole idea I had here was to make most of the content in the docs auto-generated to ensure accuracy and consistency of formatting...